todo_app/app.py

```diff
@@ -1,11 +1,31 @@
-from flask import Flask
-
+#mydict = dict((rows[0],rows[1]) for rows in reader)
+from flask import Flask, render_template, redirect, request
+from todo_app.data.session_items import get_items, add_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
+#@app.route('/')
+#def index():
+    # Modify the index() function to get the list of items 
+ #  items = get_items() 
+  # return render_template('index.html', ls = items)
 
+@app.route('/', methods = ['GET', 'POST'])
+def post_item():
+   items = get_items()
+   if request.method == 'POST':
```

Comment:
Combining these two methods can make your code more confusing in the long run, if you want to separate them out using the same address Flask lets you do so, like:
```python

@app.route('/', methods = ['GET'])
def index():
   items = get_items()
   return render_template('index.html', ls = items) 

@app.route('/', methods=["POST"])
def post_item():
   title =  request.form.get('name')
   items = add_item(title)
   print(items)
   return redirect(request.url)
```